packages/cloud-cognitive/src/components/NonLinearReading/_non-linear-reading.scss

```diff
@@ -48,10 +52,13 @@
 
 // Other Carbon settings.
 // TODO: @import 'carbon-components/scss/globals/grid/grid'; if needed
+
 // NonLinearReading uses the following Carbon components:
-// TODO: @import(s) of Carbon component styles used by NonLinearReading
+// ChevronDown16
```

Comment:
Usually reserve this for Carbon components and not icons since this section is for importing their styles.